src/main/java/dev/jbang/cli/Run.java

```diff
@@ -40,6 +40,15 @@ boolean enableFlightRecording() {
 			"--debug" }, fallbackValue = "4004", parameterConsumer = DebugFallbackConsumer.class, description = "Launch with java debug enabled on specified port (default: ${FALLBACK-VALUE}) ")
 	String debugString;
 
+	// should take arguments for package/classes when picocli fixes its flag
+	// handling bug in release 4.6.
+	// https://docs.oracle.com/cd/E19683-01/806-7930/assert-4/index.html
+	@CommandLine.Option(names = { "--enableassertions", "--ea" }, description = "Enable assertions")
+	boolean enableAssertations;
```

Comment:
assertations or assertions? :-)

(same comment because GH said it was outdated)

Reply:
at least I was consistently bad at spelling or copy/pasting ;)